test/tests.c

```diff
@@ -336,6 +336,23 @@ int test_object_key(void) {
   return 0;
 }
 
+int test_root(void) {
+	const char *js;
+	js = "\"hello\"";
+	check(parse(js, 1, 1,
+				JSMN_STRING, "hello"));
```

Comment:
For JSMN_STRING parse() expects 2 arguments, second being size, as in number of children. Just needs a 0 as argument after "hello".

Reply:
thanks! fixed it on the branch.